docs/external-node.md

```diff
@@ -0,0 +1,305 @@
+# ExternalNode
+
+## Table of Contents
+
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Name and namespace](#name-and-namespace)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+- [Security](#security)
+- [Traffic model](#traffic-model)
+  - [OVS pair ports](#ovs-pair-ports)
+  - [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## What is ExternalNode?
+
+`ExternalNode` is a CRD API that manages the network connectivity and security on Non Kubernetes Node (like a virtual
+machine or a bare-metal server). It supports specifying which network interfaces on the external Node are expected to be
+guarded with Antrea NetworkPolicy rules. The virtual machine or bare-metal server presented as `ExternalNode` can be
+either Linux or Windows. "external Node"
+will be used to represent such a virtual machine or a bare-metal server in this document.
+
+Antrea NetworkPolicy is applied to `ExternalNode` via the `ExternalEntity` resource.
```

Comment:
Could you wrap lines at a fixed column (80 to be consistent with other docs).

Reply:
updated.